crates/wasmtime/src/runtime/externals/table.rs

```diff
@@ -222,13 +216,32 @@ impl Table {
     ///
     /// Panics if `store` does not own this table.
     pub fn set(&self, mut store: impl AsContextMut, index: u64, val: Ref) -> Result<()> {
-        let store = store.as_context_mut().0;
-        let ty = self.ty(&store);
-        let val = val.into_table_element(store, ty.element())?;
-        let (table, _) = self.wasmtime_table(store, iter::empty());
-        table
-            .set(index, val)
-            .map_err(|()| anyhow!("table element index out of bounds"))
+        self.set_(store.as_context_mut().0, index, val)
+    }
+
+    pub(crate) fn set_(&self, store: &mut StoreOpaque, index: u64, val: Ref) -> Result<()> {
```

Comment:
Here an elsewhere: we have already been using the convention `foo` for the generic version and `_foo` for the opaque version, not `foo_` for the opaque version. Should be consistent and have underscores at the front.

Reply:
Ah yeah good point we should fix everything at once in a follow up.